lib/xenops_server.ml

```diff
@@ -1597,6 +1602,7 @@ and perform ?subtask ?result (op: operation) (t: Xenops_task.t) : unit =
 			debug "VM.receive_memory %s power_state = %s" id (state.Vm.power_state |> rpc_of_power_state |> Jsonrpc.to_string);*)
 
 			(* set up the destination domain *)
+			debug "VM.receive_memory creating domain and restoring VIFs";
```

Comment:
Have these log messages enough context such that we know which VM and domain we are dealing with when reading the logs? I would also ask that about the log messages below.